projects/event-lambdas/src/event-api-lambda/index.ts

```diff
@@ -30,7 +30,8 @@ async function initialise(): Promise<AppConfig> {
     (acc, curr) => (curr.value ? acc.concat([curr.value]) : acc),
     [] as string[]
   );
-
+  const LOCAL_PROFILE = 'workflow';
+  const { fromIni, fromNodeProviderChain } = require('@aws-sdk/credential-providers');
```

Comment:
I think we can use the `import` statements (like those at the top of this file) to import `fromIni` and `fromNodeProviderChain`.